elmslie-android/src/main/java/vivid/money/elmslie/android/ElmStoreLazy.kt

```diff
@@ -67,7 +67,7 @@ fun <
     )
 
 @MainThread
-internal fun <
+fun <
```

Comment:
why deinternalized?

Reply:
by accident) had some experiments, I'll fix it